signaturit-sdk.gemspec

```diff
@@ -15,7 +15,11 @@ Gem::Specification.new do |s|
 
     s.license     = 'MIT'
 
-    s.add_dependency 'rest-client', '~> 1.7'
+    s.add_dependency 'rest-client', '> 1.7'
```

Comment:
Maybe should be `>=` instead of `>`. The idea is allow use newer versions of rest-client on projects where signaturit is a dependency.